live-examples/js-examples/date-getutcmonth.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js">var date1 = new Date('December 31, 1975, 23:15:30.123 GMT+11:00');
+var date2 = new Date('December 31, 1975, 23:15:30.123 GMT-11:00');
+
+// December
+console.log(date1.getUTCMonth());
+// expected output: 11
+
+// January
+console.log(date2.getUTCMonth());
+// expected output: 0
```

Comment:
I get `NaN` for  both of these.

Reply:
Same issue with the millisecond definition in Firefox and Safari. Will fix.